### PR TITLE
Feature mobile menu

### PR DESCRIPTION
**Pull request by Leboroz**
# Feature mobile menu
## added the following features:
- When the user clicks (or taps) the hamburger button, the mobile menu appears.
- When the user clicks (or taps) the close (X) button, the mobile menu disappears.
- When the user clicks (or taps) any of the mobile menu options, the mobile menu disappears.
- When the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -25,19 +28,23 @@
...
         />
-        <button class="hamburger">
-          <img src="./images/Icons/menu-icon.png" alt="menu icon" />
+        <button id="toggleButton" class="hamburger">
+          <span class="fas fa-bars"></span>
         </button>
+        <button id="close" class="hamburger hide">
+          <span class="fas fa-times"></span>
+        </button>
+
```
Comment (safafa) [inline index.html:47]: - [ ] In the mobile view the menu Icon is missing. Kindly fix the issue.
image

**Review by safafa — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUESTED ♻️ 

Hi team, You did a great job so far 👏🏼 👏🏼 

- Well done on setting up Javascript linters;
- the linter checks are all passing;
- The desktop version of  the navbar is working as expected

However, here are some points that you need to work on to get your project approved.

## Required changes :

Check the inline comments.

## Optional changes :
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: index.html (hunk 1 above)
Comment (Leboroz) [reply index.html:47]: hello miss we are using a dependency called fontawesome.

Comment (Leboroz) [reply index.html:47]: please run npm install to view the icons

File: index.html [hunk 2]
```diff
@@ -25,19 +28,23 @@
     <header class="profile">
       <nav class="profile-menu">
```
Comment (Alexoid1) [inline index.html:29]: - [ ] Like the previous reviewer said the hamburger button and the x button is missing in the navbar kindly add those buttons.
- [ ] For mobile devices, the navbar has to overlay the whole size of the screen when the hamburger button is press.
Captura de pantalla_2021-10-04_18-04-26
- As you may see at the top there is a space that hasn't be cover.

- [ ] Kindly stick to the Figma design. Some margins, background images, and icons are missing.
**Figma**
Captura de pantalla_2021-10-04_18-10-04
- [ ] `When the user clicks (or taps) the close (X) button, the mobile menu disappear.`
- [ ] `When the user clicks (or taps) any option of the mobile menu, the mobile menu disappear.`

**Review by Alexoid1 — CHANGES_REQUESTED** [review]
# Hi @studen_name,

Impressive project!
You are doing a great job so far, and the project is looking good.
- No linter errors.
- Descriptive README.MD file.
- Follow Git-hub flow.

However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes :recycle:
 - For the project to get approved, also implement the changes mentioned below.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by carlos-ssh — APPROVED** [review]
Hi @Obote , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**